docs/doxygen/pages.dox

```diff
@@ -16,6 +23,15 @@
 All functions in the SigV4 library operate only on the buffers provided and use only
 local variables on the stack.
 </p>
+
+<h3>Compliance & Coverage</h3>
+
+<p>
+The SIGV4 library is designed to be compliant with ISO C90 and MISRA C:2012.
+All functions are written to have minimal complexity. Unit tests and CBMC proofs
```

Comment:
Is this wording the same as other libraries? "Minimal complexity" seems kind of vague

Reply:
Yeah this is same as other spoke repos. I took reference from coreJSON Library.